### ISSUE TITLE
After upgrade to 2.14.2: page is not rendered anymore

### ISSUE THREAD

**Issue by Schnodderbalken** [ember-cli/ember-cli#7394]
After having upgraded the version of ember to 2.14.2 there is a strange behavior: the page loads but the DOM does not reflect the components. There is just an empty page. The body looks like this:

```
<body class="ember-application">
    <script src="/assets/vendor.js"></script>
    <script src="/assets/filehero-frontend.js"></script>
</body>
```

There is not a single error in the console so I don't even know where to start. 

router.js (although I haven't changed anything since the upgrade):

```
import Ember from 'ember';
import config from './config/environment';

const Router = Ember.Router.extend({
    location: config.locationType,
    rootURL: config.rootURL
});

Router.map(function() {
  this.route('inbox');
  this.route('search', function() {
      this.route('index', { path: '/' });
      this.route('results', { path: '/:search_phrase' });
  });
  this.route('smartgroups', function() {
    this.route('index', { path: '/' });
    this.route('details', { path: '/details/:smartgroup_id' });
    this.route('edit', { path: '/edit/:smartgroup_id' });
    this.route('create', { path: '/create' });
    this.route('categories', function() {
      this.route('create');
      this.route('edit', { path: '/edit/:category_id' });
    });
  });
  this.route('loading');
  this.route('document-view', { path: 'document/:document_id' });
  this.route('/', {path: "*path"});
  this.route('usersettings');
});

export default Router;
```

package.json:

```
{
  "name": "filehero-frontend",
  "version": "0.0.0",
  "description": "Small description for filehero-frontend goes here",
  "license": "MIT",
  "author": "",
  "directories": {
    "doc": "doc",
    "test": "tests"
  },
  "repository": "",
  "scripts": {
    "build": "ember build",
    "start": "ember server",
    "test": "ember test"
  },
  "devDependencies": {
    "broccoli-asset-rev": "^2.4.5",
    "ember-ajax": "^3.0.0",
    "ember-cli": "~2.14.0",
    "ember-cli-app-version": "^3.0.0",
    "ember-cli-babel": "^6.3.0",
    "ember-cli-dependency-checker": "^1.3.0",
    "ember-cli-eslint": "^3.0.0",
    "ember-cli-htmlbars": "^2.0.1",
    "ember-cli-htmlbars-inline-precompile": "^0.4.3",
    "ember-cli-inject-live-reload": "^1.4.1",
    "ember-cli-qunit": "^4.0.0",
    "ember-cli-shims": "^1.1.0",
    "ember-cli-sri": "^2.1.0",
    "ember-cli-uglify": "^1.2.0",
    "ember-data": "~2.14.3",
    "ember-export-application-global": "^2.0.0",
    "ember-load-initializers": "^1.0.0",
    "ember-resolver": "^4.0.0",
    "ember-source": "~2.14.0",
    "ember-welcome-page": "^3.0.0",
    "loader.js": "^4.2.3",
    "socket.io-client": "^2.0.3",
    "ember-browserify": "^1.2.0",
    "ember-cli-sass": "5.6.0"
  },
  "engines": {
    "node": "^4.5 || 6.* || >= 7.*"
  },
  "private": true
}
```

Output from `ember version --verbose && npm --version && yarn --version`:
```
ember version --verbose && npm --version && yarn --version
ember-cli: 2.14.2
http_parser: 2.7.0
node: 6.4.0
v8: 5.0.71.60
uv: 1.9.1
zlib: 1.2.8
ares: 1.10.1-DEV
icu: 57.1
modules: 48
openssl: 1.0.2h
os: linux x64
3.10.3
bash: yarn: command not found
```

**Comment by ro0gr** [comment]
I had a similar issue when upgraded.
Do you have a "start()" invocation in your test-helper like here?

https://github.com/ember-cli/ember-cli/blob/master/blueprints/app/files/tests/test-helper.js

**Comment by Schnodderbalken** [comment]
Yes, my test-helper.js looks exactly like that.
Is there any kind of debug output I can get? It's frustrating that everything seems to be okay according to the console but the page is not rendered anymore.

**Comment by ro0gr** [comment]
I'm not aware of any. It's hard to reason about without a reproduction. 

Also it may worth to ask in [ember-cli slack](https://embercommunity.slack.com/messages/C045BNHAP/) channel.

**Comment by Turbo87** [comment]
@Schnodderbalken what dependencies did you upgrade between it working and it not working? you could try to update the dependencies one-by-one and see which dependency exactly is causing the problem.

**Comment by Schnodderbalken** [comment]
Oh, I'm sorry I haven't answered since I found the solution.
I am using websockets in my project in order to call the backend. https://github.com/thoov/ember-websockets which I had used was not compatible with the current version of ember. Inside the route where I fetch the model, this websocket connection is used, that's why I couldn't see any error or something.
I replaced the thin layer of ember-websockets with my own implementation and now everything's finde. Thank you for asking again @Turbo87 - I will close the topic.
